ideascube/search/tests/test_models.py

```diff
@@ -97,6 +97,18 @@ def test_search_Document_multiple_tag():
 
 
 @pytest.mark.usefixtures('cleansearch')
+def test_search_Document_on_tag_name_and_slug():
+    doc1 = DocumentFactory(tags=["aé"])
+    doc2 = DocumentFactory(tags=["ae"])
+
+    assert doc1 in Search.search(tags__match=["ae"])
+    assert doc2 in Search.search(tags__match=["ae"])
```

Comment:
See, I had no idea this would actually work. 😆 

I'm glad I suggested it though, and you proved it did. Thanks!